settings/default/network.lua

```diff
@@ -31,4 +31,79 @@ xi.settings.network =
     -- Central message server settings (ensure these are the same on both all map servers and the central (lobby) server
     ZMQ_IP   = "127.0.0.1",
     ZMQ_PORT = 54003,
+
+    -- ===========================
```

Comment:
Would you mind adding an additional comment block around here, saying that the "below settings are advanced options and it is very unlikely that users will need to modify them", or something to that effect?

Once these are in plainer view of regular users, I have no doubt that people will start to fiddle with them and start coming to us with all sorts of evil networking issues

Reply:
That's a good call